src/main/api/2/studio-api-2.yaml

```diff
@@ -1321,6 +1356,85 @@ components:
         gitPrivateKey:
           type: string
           description: 'Git Private Key (not returned to caller)'
+    SearchParams:
+      type: object
+      properties:
+        query:
+          type: string
+          description: The term to search for in the files
+        offset:
+          type: integer
+          format: int64
+          description: The offset to paginate the results (defaults to 0)
+        limit:
+          type: integer
+          format: int64
+          description: The limit to paginate the results (defaults to 10)
+        sortBy:
+          type: string
+          description: The name of the field to sort the results (defaults to sort by relevance)
+        sortOrder:
+          type: string
+          description: The order to sort the results (defaults to DESC)
+        filters:
+          type: object
+          description: Map containing field names and values to filter the results
```

Comment:
Please explain how these are populated. Per our discussion, the caller may use the returned facets.